src/main/java/com/minecolonies/coremod/commands/ActionMenuType.java

```diff
@@ -0,0 +1,300 @@
+            CheckForAutoDeletesCommand.class,
+            new ActionArgument("confirmDelete", ActionArgumentType.BOOLEAN, ActionArgumentType.Is.Optional)
+            )),
+    WhoAmI(new ActionMenu(
+            "WhoAmI",
+            "whoami",
+            ForgePermissionNodes.WhoAmI,
+            WhoAmICommand.class
+            )),
+    WhereAmI(new ActionMenu(
+            "WhereAmI",
+            "whereami",
+            ForgePermissionNodes.WhereAmI,
+            WhereAmICommand.class
+            )),
+
+    ListColonies(new ActionMenu(
+            "ListColonies",
+            "list",
+            ForgePermissionNodes.ListColonies,
+            ListColoniesCommand.class,
+            new ActionArgument("page", ActionArgumentType.INTEGER, ActionArgumentType.Is.Optional),
+            new ActionArgument("abandonedSinceTimeInHours", ActionArgumentType.INTEGER, ActionArgumentType.Is.Optional)
+            )),
+    RequestSystemResetAll(new ActionMenu(
+            "RequestSystemResetAll",
+            "rsResetAll",
+            ForgePermissionNodes.RequestSystemResetAll,
+            RSResetAllCommand.class
+            )),
+
+    ListCitizens(new ActionMenu(
+            "ListCitizens",
+            "list",
+            ForgePermissionNodes.ListCitizens,
+            ListCitizensCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required),
+            new ActionArgument("page", ActionArgumentType.INTEGER, ActionArgumentType.Is.Optional)
+        )),
+    KillCitizen(new ActionMenu(
+            "KillCitizen",
+            "kill",
+            ForgePermissionNodes.KillCitizen,
+            KillCitizenCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required,
+                    new ActionArgument("citizen", ActionArgumentType.Citizen, ActionArgumentType.Is.Required))
+        )),
+    RespawnCitizen(new ActionMenu(
+            "RespawnCitizen",
+            "respawn",
+            ForgePermissionNodes.RespawnCitizen,
+            RespawnCitizenCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required,
+                    new ActionArgument("citizen", ActionArgumentType.Citizen, ActionArgumentType.Is.Required))
+        )),
+    CitizenInfo(new ActionMenu(
+            "Info",
+            "info",
+            ForgePermissionNodes.CitizenInfo,
+            CitizenInfoCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required,
+                    new ActionArgument("citizen", ActionArgumentType.Citizen, ActionArgumentType.Is.Required))
+        )),
+
+    ShowColonyInfo(new ActionMenu(
+            "ShowColonyInfo",
+            "info",
+            ForgePermissionNodes.ShowColonyInfo,
+            ShowColonyInfoCommand.class,
+            // TODO: need OR() ActionArgumentType
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)